ecl/data_source_ecl_network_fic_gateway_v2.go

```diff
@@ -0,0 +1,126 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/nttcom/eclcloud/ecl/network/v2/fic_gateways"
+)
+
+func dataSourceNetworkFICGatewayV2() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNetworkFICGatewayV2Read,
+
+		Schema: map[string]*schema.Schema{
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+			"fic_service_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+			"fic_gateway_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+			"qos_option_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"tenant_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceNetworkFICGatewayV2Read(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	networkClient, err := config.networkV2Client(GetRegion(d, config))
```

Comment:
全体的にもう少し短い名前を使用した方が良いと思います。
```suggestion
	client, err := config.networkV2Client(GetRegion(d, config))
```